lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise.rb

```diff
@@ -1,17 +1,71 @@
 # frozen_string_literal: true
 
 module Kitchen::Directions::BakeInjectedExercise
-  def self.v1(exercise:)
-    V1.new.bake(exercise: exercise)
+  def self.v1(exercise:, alphabetical_multiparts: false)
+    V1.new.bake(exercise: exercise, alphabetical_multiparts: alphabetical_multiparts)
   end
 
   class V1
-    def bake(exercise:)
+    def bake(exercise:, alphabetical_multiparts:)
       question_count = exercise.injected_questions.count
       exercise[:'data-question-count'] = question_count
       exercise[:'data-is-multipart'] = question_count > 1 ? 'True' : 'False'
 
       context = exercise&.exercise_context
+      stimulus = exercise&.first("div[data-type='exercise-stimulus']")
+
+      # To handle alphabetical multipart questions without interrupting numbering
+      if alphabetical_multiparts && stimulus
+        solutions_clipboard = Kitchen::Clipboard.new
+        questions_clipboard = Kitchen::Clipboard.new
+        alphabet = *('a'..'z')
```

Comment:
Will this hold across translations? is there a more elegant way to do this?

Reply:
i believe other stuff that needs a-z is handled in a similar way ([example](https://github.com/openstax/cookbook/blob/ec192c514e4e96bd9e5767b99b852c017dc02772/lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise_question.rb#L33)). i think we had the conversation about translations at the time, but couldn't come up with a reasonable example of it becoming an issue, even down the line. it's good to note, though!